ui/apps/platform/src/Containers/NetworkGraph/simulation/SimulationFrame.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { ScreenIcon } from '@patternfly/react-icons';
+
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { NetworkPolicySimulator } from '../hooks/useNetworkPolicySimulator';
+
+type SimulationFrameProps = {
+    simulator: NetworkPolicySimulator;
+    children: React.ReactNode;
+};
+
+function SimulationFrame({ simulator, children }: SimulationFrameProps) {
+    const isSimulating =
+        simulator.state === 'GENERATED' ||
+        simulator.state === 'UNDO' ||
+        simulator.state === 'UPLOAD';
+
+    let style = {};
+    if (isSimulating) {
+        style = { position: 'relative', border: '5px solid rgb(115,188,247)' };
+    } else {
+        style = {};
+    }
+    return (
+        <div className="pf-ri__topology-demo" style={style}>
+            {children}
+            {isSimulating && (
+                <Flex
+                    className="pf-u-p-sm"
+                    style={{
+                        backgroundColor: 'rgb(224,233,242)',
```

Comment:
@alanonthegit We used to be able to inspect Invision for things like color values, or we'd use standard ones. Let's make a new year's resolution to get our color management back in shape. 🆕 📆

Reply:
I'll sync with Alan on some of these color values